server/package.json

```
@@ -0,0 +1,18 @@
+{
+  "dependencies": {
+    "json-server": "^0.17.0"
+  },
+  "name": "server",
+  "version": "1.0.0",
+  "main": "index.js",
+  "devDependencies": {
+    "json-sever": "^0.17.0"
```

오타가 있네요~ 혹시 수작업으로 작성하신 걸까요?
(오타가 있어서 npm i 했을 때 설치에 실패하네요)

패키지 설치 시 npm 커맨드라면 `--save-dev`, yarn 커맨드라면 `-D` 옵션을 붙여주면 알아서 package.json의 devDependencies에 들어간답니다!

```suggestion
    "json-server": "^0.17.0"
```